From 305bf19e390aebdf2d20ea052a92f782e8d1185c Mon Sep 17 00:00:00 2001
From: Thomas Arnhold <thomas@arnhold.org>
Date: Sun, 24 Feb 2013 17:39:08 +0000
Subject: loplugin: unused variables

And some fallout.

Change-Id: I8dfc12f3566e72d79d4f9930284c92577aa485a1
---
(limited to 'sd/source/ui/toolpanel/controls/MasterPageObserver.cxx')

diff --git a/sd/source/ui/toolpanel/controls/MasterPageObserver.cxx b/sd/source/ui/toolpanel/controls/MasterPageObserver.cxx
index 1d27c9d..cd7df65 100644
--- a/sd/source/ui/toolpanel/controls/MasterPageObserver.cxx
+++ b/sd/source/ui/toolpanel/controls/MasterPageObserver.cxx
@@ -226,8 +226,6 @@ void MasterPageObserver::Implementation::AddEventListener (
         // Tell the new listener about all the master pages that are
         // currently in use.
         typedef ::std::vector<String> StringList;
-        StringList aNewMasterPages;
-        StringList aRemovedMasterPages;
         MasterPageContainer::iterator aDocumentIterator;
         for (aDocumentIterator=maUsedMasterPages.begin();
              aDocumentIterator!=maUsedMasterPages.end();
--
cgit v0.9.0.2-2-gbebe
